compiler/rustc_session/src/session.rs

```diff
@@ -899,23 +899,24 @@ impl Session {
         ret
     }
 
+    /// Is this edition 2015?
     pub fn rust_2015(&self) -> bool {
-        self.edition() == Edition::Edition2015
+        self.edition().rust_2015()
     }
 
     /// Are we allowed to use features from the Rust 2018 edition?
     pub fn rust_2018(&self) -> bool {
-        self.edition() >= Edition::Edition2018
+        self.edition().rust_2018()
     }
```

Comment:
Wait, so `rust_2015` is `edition == 2015`, but `rust_2018` is `edition >= 2018`? That's a bit confusing IMO

Reply:
Maybe we could rename `rust_2015`.. or rename all the subsequent ones to `at_least_rust_2018`? idk